packages/mgt-components/src/components/mgt-file/mgt-file.ts

```diff
@@ -342,13 +342,14 @@ export class MgtFile extends MgtTemplatedTaskComponent {
     }
 
     let fileIconSrc;
+    let fileType: string;
```

Comment:
```suggestion
    let fileType: = "";
```
This is only assigned once so to avoid throwing errors on `undefined` values, initialize it to an empty string.